dsl/metadata.go

```diff
@@ -12,7 +12,32 @@ import (
 // services and API definitions.
 //
 // While keys can have any value the following names are handled explicitly by
-// goa when set on attributes.
+// goa when set on attributes or types.
+//
+// type:generate:force specified on a Type will generate a Go struct in the
+// service packages specified in the value. If no services are specified, it
+// generates the type in all the services in the design package. The Type
+// definition to force generate can exist in an external package and must be
+// imported in the API design.
+//
+//        package my_package
```

Comment:
I don't think we need to use this case as an example. The fact is you can import design packages at any time and this has no bearing on this feature. I would just use a simple example here.